spec/mailers/previews/customer_mailer_preview.rb

```diff
@@ -32,12 +32,26 @@ def invoice_email
   end
 
   def stolen_bike_alert_email
-    customer_contact = CustomerContact.last
+    customer_contact =
+      CustomerContact
+        .includes(:bike)
+        .limit(50)
+        .where.not(info_hash: {})
```

Comment:
Is the need to do this based on not having migrated data locally? If you pull a current version of the database is it still necessary?

Reply:
Nope — just pulled a fresh prod clone. We don't enforce referential integrity in the db, so some customer_contact records reference bikes that don't exist

Reply:
Simplified the query a bit in 23a9d1be